### DISCUSSION TITLE
Most efficient way to migrate from one Raspi OS version to newer one?

### DISCUSSION THREAD

**Discussion by gaitskell** [aaronwmorris/indi-allsky#1478] [Q&A]
I would like to update to a new version of RaspberryPi OS (Bookworm) for the hardware that I am running indi-allsky on.

Are there instructions on how to do this most efficiently - keeping indi-allsky configured identically? I see the FAQ
    https://github.com/aaronwmorris/indi-allsky/wiki/Backup-and-Recovery 
but is there a discussion on the most efficient way of migrating from one system to the next (or do those instructions cover everything for the migration)?

 My thanks, Rick

**Comment by aaronwmorris** [comment]
There are ways to upgrade Debian systems, but I am seeing information that it was not supported for Raspbian, which is unfortunate.

The backup and recovery page covers what would need to be moved.  I have just added a new section to the bottom of the Backup and Recovery page to provide step-by-step instructions on how to migrate.  I can provide clarifications if needed.
